aiida/common/pluginloader.py

```diff
@@ -125,6 +125,48 @@ def get_plugin(category, name):
 
     return plugin
 
+def load_plugin_safe(base_class, plugins_module, plugin_type):
+    """
+    It is a copy of load_plugin function to return closely related node class
```

Comment:
I would not say it is a copy, I would say it is a 'wrapper' and mention that it does not raise exceptions